get/get.go

```diff
@@ -33,38 +33,77 @@ type Cmd struct {
 	ServiceConnection   serviceConnectionCmd  `cmd:"" group:"networking.nine.ch" name:"serviceconnection" aliases:"sc" help:"Get a ServiceConnection."`
 }
 
+type output struct {
+	Format        outputFormat `help:"Configures list output. ${enum}" name:"output" short:"o" enum:"full,no-header,contexts,yaml,stats,json" default:"full"`
+	AllProjects   bool         `help:"apply the get over all projects." short:"A"`
+	AllNamespaces bool         `help:"apply the get over all namespaces." hidden:""`
+	Watch         bool         `help:"Watch resource(s) for changes and print the updated resource." short:"w"`
+	tabWriter     *tabwriter.Writer
+	writer        io.Writer
+}
+
 type resourceCmd struct {
 	Name string `arg:"" predictor:"resource_name" help:"Name of the resource to get. If omitted all in the project will be listed." default:""`
 }
 
-type output string
+type outputFormat string
 
 const (
-	full     output = "full"
-	noHeader output = "no-header"
-	contexts output = "contexts"
-	yamlOut  output = "yaml"
-	stats    output = "stats"
-	jsonOut  output = "json"
+	full     outputFormat = "full"
+	noHeader outputFormat = "no-header"
+	contexts outputFormat = "contexts"
+	yamlOut  outputFormat = "yaml"
+	stats    outputFormat = "stats"
+	jsonOut  outputFormat = "json"
 )
 
-func (cmd *Cmd) list(ctx context.Context, client *api.Client, list runtimeclient.ObjectList, opts ...api.ListOpt) error {
+func (cmd *Cmd) AfterApply() error {
+	cmd.initOut()
+	return nil
+}
+
+type listPrinter interface {
+	print(context.Context, *api.Client, runtimeclient.ObjectList, *output) error
+	list() runtimeclient.ObjectList
+}
+
+func (cmd *Cmd) list(ctx context.Context, client *api.Client, lp listPrinter, opts ...api.ListOpt) error {
```

Comment:
Should this be renamed to something like `listPrint` since it takes care of both now?